.buildkite/scripts/steps/integration-package.sh

```diff
@@ -2,5 +2,11 @@
 set -euo pipefail
 
 source .buildkite/scripts/common.sh
+
 # Remove AGENT_PACKAGE_VERSION pinning as soon as 9.0.0 is released
-AGENT_PACKAGE_VERSION=9.0.0 PACKAGES=tar.gz,zip,rpm,deb PLATFORMS=linux/amd64,linux/arm64,windows/amd64  SNAPSHOT=true EXTERNAL=true  mage package
+export AGENT_PACKAGE_VERSION=9.0.0
+
+export SNAPSHOT="true"
+export EXTERNAL="true"
+
+mage package
```

Comment:
What's the reason for using exports instead of the former approach with in-line variables?

```suggestion
AGENT_PACKAGE_VERSION=9.0.0 SNAPSHOT=true EXTERNAL=true mage package
```

Reply:
to improve readability, since previously there were a lot of params and command was too long